src/pages/auth/signup.tsx

```diff
@@ -168,6 +168,24 @@ const Page: NextPage = () => {
             <FormErrorMessage>{errors.password?.message}</FormErrorMessage>
           </FormControl>
 
+          <FormControl isRequired isInvalid={}>
+            <FormLabel htmlFor={"password"}>Confirm Password</FormLabel>
+
+            <Controller
+              control={control}
+              name={"password"}
```

Comment:
Please change the name of the field from `password` to `confirmPassword`, since this will process the invalid value passed for the password field.

Reply:
The other one is fine, but changing this results in "Type '"confirmPassword"' is not assignable to type '"username" | "lastName" | "firstName" | "email" | "password"'. I assume you created a custom union type, where is that declared?